packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/Protocols/MethodChannelProtocol.h

```diff
@@ -0,0 +1,22 @@
+#if TARGET_OS_OSX
+#import <FlutterMacOS/FlutterMacOS.h>
+#else
+#import <Flutter/Flutter.h>
+#endif
+
+NS_ASSUME_NONNULL_BEGIN
+@protocol MethodChannel <NSObject>
+// Invokes the specified Flutter method with the specified arguments, expecting
+// an asynchronous result.
+- (void)invokeMethod:(NSString *)method arguments:(id _Nullable)arguments;
+- (void)invokeMethod:(NSString *)method
+           arguments:(id _Nullable)arguments
+              result:(FlutterResult _Nullable)callback;
+@end
+
+@interface DefaultMethodChannel : NSObject <MethodChannel>
+@property(strong, nonatomic) FlutterMethodChannel *channel;
```

Comment:
this property should be in .m file

Reply:
why? it should be part of the interface

Reply:
Because it is the implementation detail. 

You can have an interface in .m file. That interface will be a class extension.